### ISSUE TITLE
unstable whan opening several notebooks 0.74.2 win 10 

### ISSUE THREAD

**Issue by osman1963** [zim-desktop-wiki/zim-desktop-wiki#1786]
**Describe the bug**
After opening a notebook and trying to open a second one it doesn't fully open. It appears when hovering on Zim icon in the tray but doesn't fully open , even if I click on it's small window in the tray. This happens sometimes with the third or fourth notebook, and sometimes even with the second. It Starts this behaviour after about a week of use. Uinstalling and removing AppData of the app and reinstalling it solves the problem, but I can't re-install it every week or so. Restarting the computer doesn't solve it. Is there something I should be aware of? change in some way? Any help is really appreciated.
I'm using version 0.74.2 on Windows 10

**Comment by jaap-karssenberg** [comment]
Haven't observed that behavior myself and haven't heard a report before.

I think uninstall doesn't do anything. Probably clearing AppData is what
makes the difference. Would be interesting to know what file in AppData
exactly to narrow down the cause.

-- Jaap

**Comment by osman1963** [comment]
I removed the whole zim folder from appdata, then re-installed.
Last thing I did yesterday, was to uninstall, remove zim folder from
appdata, then installed the portable version of zim. Just after
installation everything worked fine, but this morning I couldn't open but
one notebook only.
Could this be related to some limitations in python or qt related to the
fact that I'm using an international keyboard (4 languages, one of them is
rtl, Arabic). I'm saying this because I also faced an inexplicable problem
with OneMore add-on used with OneNote, and it turned out that the API had a
warning against using some commands with international keyboards, and
recommendations to use other ones. So I just thought it might be a similar
case here too.
Thanks for your prompt response

**Comment by jaap-karssenberg** [comment]
No, these kinds of intermittent issues don't seem related to
internationalization.

BTW. Zim is using GTK, not QT. And pretty sure one note is not GTK based
either.

Are you using version control of some kind, it other background processes
that might lock up?

Op wo 8 dec. 2021 13:51 schreef osman1963 ***@***.***>:

> I removed the whole zim folder from appdata, then re-installed.
> Last thing I did yesterday, was to uninstall, remove zim folder from
> appdata, then installed the portable version of zim. Just after
> installation everything worked fine, but this morning I couldn't open but
> one notebook only.
> Could this be related to some limitations in python or qt related to the
> fact that I'm using an international keyboard (4 languages, one of them is
> rtl, Arabic). I'm saying this because I also faced an inexplicable problem
> with OneMore add-on used with OneNote, and it turned out that the API had a
> warning against using some commands with international keyboards, and
> recommendations to use other ones. So I just thought it might be a similar
> case here too.
> Thanks for your prompt response
>
> On Wed, Dec 8, 2021 at 10:11 AM Jaap Karssenberg ***@***.***>
> wrote:
>
> > Haven't observed that behavior myself and haven't heard a report before.
> >
> > I think uninstall doesn't do anything. Probably clearing AppData is what
> > makes the difference. Would be interesting to know what file in AppData
> > exactly to narrow down the cause.
> >
> > -- Jaap
> >
> >

**Comment by osman1963** [comment]
How can I know if there is a background process locking up?
I started learning python lately and I'm using VSCode beside python's IDLE,
could this be related to the problem in a way?

**Comment by jaap-karssenberg** [comment]
No, it is about processes started from zim itself, e.g. when using the
version control plugin or using custom tools.

-- Jaap

**Comment by victorfeight** [comment]
Can you describe more what you mean by "doesn't fully open" and whether you're running from source or from Desktop version? If you mean, it appears in the Windows task bar but no visible screen, I have encountered this before I set up the correct launch configuration, specifically with interwiki links.

What fixed this behavior was creating a Windows shortcut with the target set as:
`C:\git-sdk-64\mingw64\bin\python3w.exe "C:\Apps\zim-0.74.3\zim.py"`

**Comment by osman1963** [comment]
What you described is exactly what was happening with me. I thought it
might be related to the version of python installed before installing zim.
I was running the Desktop app.
Accidentally I had to re-install windows, I didn't install python after
installing windows, then installed zim 0.74 and it worked fine since then.
Thanks for your help and have a nice day :)

**Comment by WS-HS** [comment]
i need say... GTK in windows is very bad...

Localization correspondence -> terrible. 
And it can be said that there has been no improvement for thousands of years. 
Users should not be expected. Can solve it on its own. 

Localization is not enabled correctly. 

Efficacy - > terrible bad Pauses, 
freezes, high CPU, almost any bad state, can all "normalize" things... 

Supportability - > Most of the QT cases, LINUX->WIN.linux needs software-dependent plug-ins. There are almost solutions. GTK is terrible...

GUI -> very poor integration with WINDOWS.
Example: Who really wants to use GTK's file management in Windows?
That's not to mention the poor file access efficiency of .GTK under Windows...

I know that moving a software from GTK to QT-win is not really that easy.

But .GTK on windows. Really cause problems. Difficult to promote to my friends. (especially non-English language and only use windows friends)
Even those friends who are more capable of using computers will give up using them because of strange problems such as freezing.

Now almost 90% of the successful cross-platform note-taking software. All choose to use QT-WIN.

I know the transfer is bound to be difficult and time consuming.
But now it's really time to think. Move to QT.

QT's cross-platform resources, with solutions. More and more. Opposite. GTK...

**Comment by jaap-karssenberg** [comment]
@WS-HS thanks for the rant, feel free to make a QT version, but don't expect me to work on that

**Comment by jaap-karssenberg** [comment]
Need better info on how to reproduce to make any progress on this issue

**Comment by osman1963** [comment]
to reproduce:
1) download the win64 version from https://zim-wiki.org/downloads/ on a win64 windows 10 machine
2) create several notebooks
3) write anything in each, just a few lines here and there
4) close everything
5) Open the app again
6) tray opening all the notebooks you created
After opening two or three notebooks it won't open any other ones
